### DISCUSSION TITLE
Assessment Plan - diagram update?

### DISCUSSION THREAD

**Discussion by paulJRCurtis** [usnistgov/OSCAL#1584] [General]
I note a discrepancy between the handy Assessment Plan diagram (as per below) and the actual model. 

The diagram shows `Assessment Action` which does not appear as a root level object. 

Is this essentially equivalent to `[local-definitions][activities]` which would be referenced from `[tasks][associated-activities]`?

![assessment-plan-model](https://user-images.githubusercontent.com/32033064/207224784-35d7906d-6264-42ff-9a34-063c4947edf5.svg)

**Comment by iMichaela** [comment]
@paulJRCurtis - correct. The model was updated before the release, but we missed updating the diagrams. We planned for a while to have  professionally done diagrams and update the colors (and use them consistently) but we never managed to do so. We regret the layers diagram on [https://pages.nist.gov/OSCAL/concepts/layer/](https://pages.nist.gov/OSCAL/concepts/layer/) has different colors than the rest of the diagrams and hope to be able to address this issue.

**Reply by paulJRCurtis** [reply]
Thanks you. 
Now that the diagrams are covering Christmas trees the world over you may have issues with backward compatibility.

**Reply by iMichaela** [reply]
Backward compatibilities for the Christmas trees was never the OSCAL team's promise :)
